mne/decoding/classifier.py

```diff
@@ -453,3 +453,23 @@ def transform(self, epochs_data, y=None):
                                      picks=self.picks, n_jobs=self.n_jobs,
                                      copy=False, verbose=False)
         return epochs_data
+
+
+def compute_patterns(epochs, linear_model):
```

Comment:
They are quite different, this code is related to the paper by Haufe et al, 2014 (neuroimage).
It computes patterns from a linear model (e.g. a RidgeClassifier) and put it into an evoked object which is more convenient for visualisation.